### DISCUSSION TITLE
Editing invitation email template

### DISCUSSION THREAD

**Discussion by rollecode** [plausible/analytics#2282] [Self-Hosted Support]
Is there a way to edit [invitation email content](https://github.com/plausible/analytics/blob/6fe032880ad44a89d8d2c5194878f3db2f7d6b45/lib/plausible_web/templates/email/new_user_invitation.html.eex) within a docker container?

**Comment by chrisk-7777** [comment]
I just wanted to +1 this, and add further context. 

Without briefing clients prior, getting an email from a stranger (my client doesn't know who Uku and Marko are), has been a bit confusing.
